docs/plugins/irods_rule_language.md

```diff
@@ -586,6 +586,9 @@ This Is Not A Valid Function Name or Rule Name
 
 ### Rule Condition
 
+!!! Note
+    **Use of `on()` may lead to unexpected behavior!** If all rule conditions for a rule fail, the rule will return an error! Use `if()` for simple conditions.
```

Comment:
The rich diff looks a little strange here. "!!! Note" is actually being printed in addition to getting the indentation with the little bar on the left. Does this look right when it's rendered in the docs?

Reply:
Rich diff?
Here's what I see:
![temp](https://github.com/irods/irods_docs/assets/127796673/db2c6ee5-572d-4acf-8858-6a6a0ab75643)

Reply:
Oh, I think the diff in github must render differently. What you showed looks better! Resolving this